build.gradle

```diff
@@ -283,7 +283,7 @@ allprojects {
         }
       }
       boolean hasShadow = project.plugins.hasPlugin(ShadowPlugin)
-      project.configurations.compile.dependencies
```

Comment:
deprecated `compile` has been removed. 
This should have been replaced before but now ultimately caused the build to fail